tests/e2e/e2e_workflow.spec.js

```diff
@@ -192,6 +192,120 @@ describe('Basic end-to-end Workflow', function () {
       });
   });
 
+  it('Installs sass support successfully via `ng install sass`', function() {
+    this.timeout(420000);
+
+    sh.exec('npm install node-sass', { silent: true });
```

Comment:
Hi @gkalpak. I need to update the description, you are right. We abandoned the `ng install` command for now.